### ISSUE TITLE
Rename the repository and fix all references

### ISSUE THREAD

**Issue by zirkov** [Roblox/chef-sc#6]
The proposed new name is `chef-sauceconnect`
Also, we need to point out in the README that all command-line parameters to `sc` should be set through policyfiles (`default['sc']['server']['cmd_options']` attribute)
The dependencies are:
* [rblx_policyfiles/policyfiles/saucelab](https://github.com/Roblox/rblx_policyfiles/tree/master/policyfiles/saucelabs):
  * both `_saucelabs_base.rb` and `sauceconnect.rb` reference `sc` cookbook

**Comment by zirkov** [comment]
@Brantone @mattcl any comments/suggestions/objections on the todo list?

**Comment by Brantone** [comment]
Yeah, that makes sense ... if anything, in future could construct a "default vs. extras" cmd options, but that's for the future.
